cegs_portal/search/tsv_templates/v1/non_targeting_reos.py

```diff
@@ -1,11 +1,42 @@
+from cegs_portal.search.templatetags.custom_helpers import format_significance
+
+
 def non_targeting_regulatory_effects(data):
     reo_page, feature = data
     tsv_data = []
-    tsv_data.append(['Chromosome Location', 'Effect Size', 'Direction', 'Significance', 'Distance', 'Feature Type', 'Experiment'])
+    tsv_data.append(
+        [
+            "chrom",
```

Comment:
IIRC Alex said something about the header line being a comment. so instead of `chrom` the first entry should be `# chrom` I think.

Reply:
He said it was fine today when I showed him. Are you saying "#" and then "chrom" or "#chrom"

Reply:
"#[space]chrom"

Reply:
Got it